### ISSUE TITLE
Feature: Adding Hovering effect on the button of more on the categories section of the Home page

### ISSUE THREAD

**Issue by Divyanshgarg-04** [Sriparno08/Openpedia#228]
Feature: Adding Hovering effect on the button of more on the categories section of the Home page.

<img width="747" alt="image" src="https://github.com/Sriparno08/Openpedia/assets/120127517/0c0a22d0-8b7d-4f06-a97e-ef6b4b612062">

**Comment by vishal2005025** [comment]
I have read the issue and I would like to work on this issue.
Please assign it to me under JWOC'2024.

**Comment by Sriparno08** [comment]
@Divyanshgarg-04 You're assigned.

@vishal2005025 You're already assigned to another issue. Once your PR for it is merged, you will be considered for other issues.

**Comment by vermaanurag1532** [comment]
please Assign it to me
